projects/1.20-fabric/assets/modrinth-lapisworks/lapisworks/lang/zh_cn.json

```
@@ -396,15 +396,15 @@
 
     "pages.lapisworks.interop.lapixtended.title": "青金装备拓展",
     "pages.lapisworks.interop.lapixtended.get_hextended_b-baka": "我能感受到我站在灵感的悬崖之前，但我就是翻不过它……$(br2)我能感觉到它是咒法学的拓展，就像是某种锦上添花的“$(thing)附属$()”。更具体地说，这个附属的主要内容，是$(o)意识的延伸的延伸$()。是咒法学的延伸。$(thing)咒法延伸$()，或者详细说来，$(thing)咒法装备拓展$()。大概是这样？无论怎么说，我会在真正翻过那个悬崖，或者说“安装附属”之后重写这些条目。",
-    "pages.lapisworks.interop.lapixtended.intro": "好多法杖！而且好好看！我必须给所有法杖都设计一个未完全灌注的版本！$(br2)$(o)我试过让黑曜石染上“泪痕”，和哭泣的黑曜石差不多，但可惜法杖太细长，泪痕在上面不够美观。$()$(br2)$(o)紫水晶法杖已经足够了。$()",
+    "pages.lapisworks.interop.lapixtended.intro": "好多法杖！而且好好看！我必须给所有法杖都设计一个未完全灌注的版本！$(br2)$(o)我试过让黑曜石染上“泪痕”，和$(item)哭泣的黑曜石$()差不多，但可惜法杖太细长，泪痕在上面不够美观。$()$(br2)$(o)紫水晶法杖已经足够了。$()",
     "pages.lapisworks.interop.lapixtended.pic.desc": "$(o)魔杖的完全灌注需要 20 个$(item)紫金石$()$(o)，但灌注后相较法杖更精确。$()",
-    "pages.lapisworks.interop.lapixtended.orb": "我没法仿造实体散发出的能量。不过，$(item)紫金石$()确实有强大的影响能力。因此，我向绘图水晶球里灌注了 10 个$(item)紫金石$()，得到了紫金校准水晶球。它能将 10 个$(item)紫金石$()转变成处在我头部位置的极微小粒子，再把它们排布成连在头部和水晶球之间不断变形的线。只要我手持水晶球，即便有区块加载的神秘力量，这条线依然能延伸得$(o)极长$()，并在设定位置处提供大约 3 格的影响范围。我应当在此说明，转变后的$(item)紫金石$()无法再变回有形的固体。",
+    "pages.lapisworks.interop.lapixtended.orb": "我没法仿造实体散发出的能量。不过，$(item)紫金石$()确实有强大的影响能力。因此，我向绘图水晶球里灌注了 10 个$(item)紫金石$()，得到了$(item)紫金校准水晶球$()。它能将 10 个$(item)紫金石$()转变成处在我头部位置的极微小粒子，再把它们排布成连在头部和水晶球之间不断变形的线。只要我手持水晶球，即便有区块加载的神秘力量，这条线依然能延伸得$(o)极长$()，并在设定位置处提供大约 3 格的影响范围。我应当在此说明，转变后的$(item)紫金石$()无法再变回有形的固体。",
     
     "pages.lapisworks.interop.lapixical.title": "Lapixical",
-    "pages.lapisworks.interop.lapixical.get_hexical_bro": "Lapixical",
+    "pages.lapisworks.interop.lapixical.get_hexical_bro": "我在探索过程中听到了有关某个物件的传说。它极为强大，却仿佛只是人为捏造的幻影。“$(thing)精灵$()”。它们有着太多的谜题和悖论。有些晦涩的卷轴称它们是许愿机，其他文献则说这都是谎言。有人说它们是被困在了神灯中，还有些人说他们真的找到了类似精灵的事物。无论如何，只要这堆让我困惑不已（也可以说很……$(thing)咒法风味（Hexical）$()的？）神话最后证实出来是谣言，我就会重写一遍这些条目。",

```

```suggestion
    "pages.lapisworks.interop.lapixical.get_hexical_bro": "在旅途中，我曾听闻过某个物件的传说，它被描绘得如此强大，以至于更像是空想。“$(thing)精灵$()”。它们有着太多的谜题和悖论。有些晦涩的卷轴称它们是许愿机，另一些记载则却直言这是谎言。有人说它们被困于神灯中，但也有人声称他们在现实世界中亲眼目睹过其族群。无论真相如何，只要这堆让我困惑不已（也可以说很……$(thing)咒法风味（Hexical）$()的？）神话最后能证实为谣言，我就会重写一遍这些条目。",
```
括号里的实在是没看懂，这都什么语法啊……